### PR TITLE
Support OmniAuth 2

### PR DESCRIPTION
**Pull request by stevenharman**
This follows #22 and #24 to get us up to OmniAuth 2.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,30 @@ use OmniAuth::Builder do
 end
 ```
 
+#### Upgrading to 1.0+
+
+Versions before `1.0` allowed you to pass `:scope` option as a block, just as today.
+However, that block received a `Rack::Request` instance, rather than the `Rack` `env`.
+If you used that given `Rack::Request` in your `:scope` block you can update your code create a new instance, from the `env`.
```
Comment (pafuent) [inline README.md:109]: s/you can update your code create a new instance/you can update your code and create a new instance/

**Review by pafuent — COMMENTED** [review]
Just a typo

File: README.md (hunk 1 above)
Comment (stevenharman) [reply README.md]: Good catch. Thank you!

**Review by pafuent — APPROVED** [review]
LGTM!

**Pull request merged**